src/main/java/io/jenkins/pluginhealth/scoring/http/ProbeResultController.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2022 Jenkins Infra
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package io.jenkins.pluginhealth.scoring.http;
+
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import io.jenkins.pluginhealth.scoring.probes.Probe;
+import io.jenkins.pluginhealth.scoring.service.PluginService;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.servlet.ModelAndView;
+
+@Controller
+@RequestMapping(path = "/results")
```

Comment:
General question: why not have this behind `/probes/results` and have the code in the `ProbesController`?

Reply:
I've been trying to do this and the `probes/results` view gets built successfully with expected results but I broke the navbar on the left. The 2 icons are not visible when:
- we're on the Probe Results page (via `/probes/results`),
- and while being on the `probes/results` page, when we delete `results` from this URL and hit ENTER.

Any hints please?

Reply:
I think it's because of the ionicons library which is not able to grab the svg file correctly.

Reply:
Okay I'll check, but the icons are visible in one view and not in the other, makes me wonder how are these icons dependent on the mapping. Before my latest commit, everything works as expected. But I didn't touch any <ion-icons> tags or *.svg files in this latest commit...

Reply:
> Okay I'll check, but the icons are visible in one view and not in the other, makes me wonder how are these icons dependent on the mapping

if you look at your "network" tab in the developer tools of the web browser you are using, you'll see that the icons are fetched from different paths depending if you are on `/probes` or `/probes/results`, which shouldn't be the case.

Reply:
Thanks!! On the "network" tab I noticed that the `probes/results` view was appending "probes" before the icon path, making it incorrect. So, I explicitly specified the icon path starting with `/` so that thymeleaf doesn't add the prefix "probes" to it. And this "probes" prefix was coming from `GetMapping("probes")`.